docker/docker-entrypoint.sh

```diff
@@ -3,19 +3,13 @@
 set -e
 
 echo "setting up the database"
-django-admin.py migrate
+django-admin.py migrate --noinput
 django-admin.py collectstatic --noinput
 
 echo "from django.contrib.auth.models import User
 if not User.objects.filter(username='admin').count():
     User.objects.create_superuser('admin', 'admin@example.com', 'pass')
 " | django-admin.py shell
 
-echo "=> Starting nginx"
-nginx
-
 echo "=> Starting Supervisord"
-supervisord -c /etc/supervisord.conf
-
-echo "=> Tailing logs"
-tail -qF /var/log/supervisor/*.log
+exec supervisord -c /etc/supervisor/supervisord.conf
```

Comment:
@miltontony see the difference:
```
> $ docker run -d --name no-exec alpine sh -c 'sleep 100'
4b10ecbf39cf5758ccc010295c6d4040d1de61fd96a0924aeca00b3105c6e942

> $ docker exec no-exec pstree -p
sh(1)---sleep(5)
```

versus:
```
> $ docker run -d --name exec alpine sh -c 'exec sleep 100'
d68a7ba1efb9854c0df212644556aa029fe74bf147e012243e1a9c55a2d8fa94

> $ docker exec exec pstree -p
sleep(1)
```

Apart from being unnecessary, the shell process ignores certain signals and doesn't pass signals on to its child processes.